internal/pkg/summary/consolidated.go

```diff
@@ -522,36 +530,78 @@ func createDir(path string, ignoreexisting bool) error {
+		return err
+	}
 
-	// create header
-	for k, v := range header {
-		_ = sheet.SetCellValue(sheeName, k, v)
+	err = cs.buildDocumentationForPlugin(PluginNameOpenShiftConformance)
+	if err != nil {
+		return err
 	}
 
 	return nil
 }
 
-// populateGsheet fill each row per error item.
-func populateSheet(sheet *excelize.File, sheeName, suite string, list []string, rowN *int64) {
-	for idx, v := range list {
-		_ = sheet.SetCellValue(sheeName, fmt.Sprintf("A%d", *rowN), suite)
-		_ = sheet.SetCellValue(sheeName, fmt.Sprintf("B%d", *rowN), idx+1)
-		_ = sheet.SetCellValue(sheeName, fmt.Sprintf("C%d", *rowN), sheeName)
-		_ = sheet.SetCellValue(sheeName, fmt.Sprintf("D%d", *rowN), v)
-		_ = sheet.SetCellValue(sheeName, fmt.Sprintf("E%d", *rowN), "TODO Review")
-		_ = sheet.SetCellValue(sheeName, fmt.Sprintf("F%d", *rowN), "")
-		*(rowN) += 1
+// applyFilterFlakeForPlugin query the Sippy API looking for each failed test
+// on each plugin/suite, saving the list on the ResultSummary.
+func (cs *ConsolidatedSummary) buildDocumentationForPlugin(plugin string) error {
+
+	var (
+		ps               *OPCTPluginSummary
+		version          string
+		docUserBaseURL   string
+		docSourceBaseURL string
+	)
+
+	switch plugin {
+	case PluginNameKubernetesConformance:
+		ps = cs.GetProvider().GetOpenShift().GetResultK8SValidated()
+		versionFull := cs.GetProvider().GetSonobuoyCluster().APIVersion
+		reVersion := regexp.MustCompile(`^v(\d+\.\d+)`)
+		matches := reVersion.FindStringSubmatch(versionFull)
+		if len(matches) != 2 {
+			log.Warnf("Unable to extract kubernetes version to build documentation: %v [%v]", versionFull, matches)
+			return nil
+		}
+		version = matches[1]
+		// fmt.Println(matches)
+		docUserBaseURL = fmt.Sprintf("https://github.com/cncf/k8s-conformance/blob/master/docs/KubeConformance-%s.md", version)
+		docSourceBaseURL = fmt.Sprintf("https://raw.githubusercontent.com/cncf/k8s-conformance/master/docs/KubeConformance-%s.md", version)
+	case PluginNameOpenShiftConformance:
+		ps = cs.GetProvider().GetOpenShift().GetResultOCPValidated()
+		// v, err := cs.GetProvider().GetOpenShift().GetClusterVersionXY()
+		// if err != nil {
+		// 	return errors.Errorf("Error getting cluster version: %v", err)
+		// }
+		// version = v
+		// OCP tests does not have documentation
+		// https://docs.openshift.com/container-platform/4.13/welcome/index.html
+		// https://access.redhat.com/search/
+		docUserBaseURL = "https://github.com/openshift/origin/blob/master/test/extended/README.md"
+		docSourceBaseURL = docUserBaseURL
+	default:
+		return errors.New("Plugin not found to apply filter: Flaky")
 	}
-}
 
-// save the excel sheet to the disk.
-func saveSheet(sheet *excelize.File, sheetFileName string) {
-	if err := sheet.SaveAs(sheetFileName); err != nil {
-		log.Error(err)
+	if ps.Documentation == nil {
+		ps.Documentation = NewTestDocumentation(docUserBaseURL, docSourceBaseURL)
+		err := ps.Documentation.Load()
+		if err != nil {
+			return err
+		}
```

Comment:
If load failed, we must log the error, but not return error and failed the execution. Considering this is not required by the execution, having tests without docs linked due to external connectivity is acceptable.